corehq/apps/app_manager/feature_support.py

```diff
@@ -12,8 +12,8 @@ class CommCareFeatureSupportMixin(object):
     def _require_minimum_version(self, minimum_version):
         if settings.UNIT_TESTING and self.build_version is None:
             return False
-        assert isinstance(self.build_version, Version)
-        assert isinstance(minimum_version, (str, Version))
+        assert isinstance(self.build_version, LooseVersion)
+        assert isinstance(minimum_version, (str, LooseVersion))
```

Comment:
Is `LooseVersion` the same as `Version`, or should this be switched to use `packaging.version.Version`?

Reply:
`LooseVersion` is in fact not the same as `Version`. We rely on several attributes present in `LooseVersion` but not in the new `packaging.version.Version` for how we utilize it in creating commcare build numbers. I thought about creating a wrapper, but the reality is we are not using `packaging.version` with its intended purpose to determine minimum build requirements for python packages. We instead use it for versioning in app builder

Reply:
Yes, it is correct that `LooseVersion` is the one that should be passed here as this code was just checking to ensure it was either a `LooseVersion` or `StrictVersion` from the previous `distutils.version` package. However, because `packaging.version.Version` and `looseversion.LooseVersion` no longer share the same base class, this code no longer makes sense